rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -3878,8 +3879,18 @@ void PPUTranslator::FMADDS(ppu_opcode_t op)
 	const auto a = GetFpr(op.fra);
 	const auto b = GetFpr(op.frb);
 	const auto c = GetFpr(op.frc);
-	const auto result = m_ir->CreateFPTrunc(m_ir->CreateFAdd(m_ir->CreateFMul(a, c), b), GetType<f32>());
-	SetFpr(op.frd, result);
+
+	llvm::Value* result;
+	if(g_cfg.core.ppu_accurate_fma)
```

Comment:
Here and everywhere:
```suggestion
	if (g_cfg.core.ppu_accurate_fma)
```